io/cmd.go

```diff
@@ -0,0 +1,185 @@
+package io
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+	"sync"
+)
+
+// Returns the output of the external process.
+// If the output is not needed, use RunCmd
```

Comment:
Suggesting to rephrase a bit:

// Executes an external process and returns its output.
// If the returned output is not needed, use the RunCmd function instead , for better performance.

Reply:
Done